### PR TITLE
ENH/WIP: start to thread configuration through

### PR DESCRIPTION
**Pull request by tacaswell**
Add the ability to pass a kafka topic / server through the cli.

### CODE REVIEW

**Comment by tacaswell** [comment]
This is needed for the code in https://github.com/bluesky/bluesky-pods/pull/14 to work

File: bluesky_queueserver/manager/start_manager.py [hunk 1]
```diff
@@ -29,6 +32,7 @@ def __init__(self):
         self._watchdog_state_lock = threading.Lock()
 
         self._manager_is_stopping = False  # Set True to stop the server
+        self.config = config
```
Comment (dmgav) [inline bluesky_queueserver/manager/start_manager.py:35]:
```suggestion
        self._config = config
```

File: bluesky_queueserver/manager/start_manager.py [hunk 2]
```diff
@@ -46,7 +50,9 @@ def _start_re_worker_handler(self):
         """
         logger.info("Starting RE Worker ...")
         try:
-            self._re_worker = RunEngineWorker(conn=self._manager_conn, name="RE Worker Process")
+            self._re_worker = RunEngineWorker(
+                conn=self._manager_conn, name="RE Worker Process", config=self.config,
```
Comment (dmgav) [inline bluesky_queueserver/manager/start_manager.py:54]:
```suggestion
                conn=self._manager_conn, name="RE Worker Process", config=self._config,
```

File: bluesky_queueserver/manager/start_manager.py [hunk 3]
```diff
@@ -145,11 +151,25 @@ def run(self):
 
 
 def start_manager():
-
+    parser = argparse.ArgumentParser(
+        description="Start a RE queueserver and worker processes",
```
Comment (dmgav) [inline bluesky_queueserver/manager/start_manager.py:155]:
```suggestion
        description="Start a RE Manager",
```

**Review by dmgav — APPROVED** [review]
Just a few minor suggestions. Did you have a chance to check it works with Kafka?

**Comment by tacaswell** [comment]
This is the code I was running on the call.

**Pull request merged**